<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageListWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>239</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/411.bmp</normaloff>:/images/icons/411.bmp</iconset>
       </attribute>
       <attribute name="label">
        <string>QListWidget操作</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QToolButton" name="tBtnListIni">
          <property name="text">
           <string>tBtnListIni</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="tBtnListInsert">
          <property name="text">
           <string>tBtnListInsert</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QToolButton" name="tBtnListAppend">
          <property name="text">
           <string>tBtnListAppend</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="tBtnListClear">
          <property name="text">
           <string>tBtnListClear</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QToolButton" name="tBtnListDelete">
          <property name="text">
           <string>tBtnListDelete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageTreeWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>200</width>
         <height>239</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/409.bmp</normaloff>:/images/icons/409.bmp</iconset>
       </attribute>
       <attribute name="label">
        <string>QTreeWidget操作</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="pageTableWidget">
       <attribute name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/images/icons/413.bmp</normaloff>:/images/icons/413.bmp</iconset>
       </attribute>
       <attribute name="label">
        <string>QTableWidget操作</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabListWidget">
       <attribute name="title">
        <string>QListWidget</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>当前项变化</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QToolButton" name="tBtnSelectItem">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>tBtnSelectItem</string>
             </property>
             <property name="popupMode">
              <enum>QToolButton::MenuButtonPopup</enum>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
             <property name="autoRaise">
              <bool>false</bool>
             </property>
             <property name="arrowType">
              <enum>Qt::RightArrow</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tBtnSelAll">
             <property name="text">
              <string>tBtnSelAll</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tBtnSelNone">
             <property name="text">
              <string>tBtnSelNone</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="tBtnSelInvs">
             <property name="text">
              <string>tBtnSelInvs</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>h</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>是否</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabTreeWidget">
       <attribute name="title">
        <string>QTreeWidget</string>
       </attribute>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>491</width>
          <height>281</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>a</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>v</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>是</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tabTableWidget">
       <attribute name="title">
        <string>QTableWidget</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>511</width>
          <height>301</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>唰唰唰</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ss</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListIni"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListAppend"/>
   <addaction name="actListDelete"/>
  </widget>
  <action name="actListIni">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/129.bmp</normaloff>:/images/icons/129.bmp</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
   <property name="toolTip">
    <string>初始化列表</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/delete2.bmp</normaloff>:/images/icons/delete2.bmp</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
   <property name="toolTip">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/425.bmp</normaloff>:/images/icons/425.bmp</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="actListAppend">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/317.bmp</normaloff>:/images/icons/317.bmp</iconset>
   </property>
   <property name="text">
    <string>添加项</string>
   </property>
   <property name="toolTip">
    <string>添加项</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/325.bmp</normaloff>:/images/icons/325.bmp</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
   <property name="toolTip">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="actSelAll">
   <property name="text">
    <string>全选</string>
   </property>
   <property name="toolTip">
    <string>全选</string>
   </property>
  </action>
  <action name="actSelNone">
   <property name="text">
    <string>全不选</string>
   </property>
   <property name="toolTip">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
   <property name="toolTip">
    <string>反选</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/exit2.bmp</normaloff>:/images/icons/exit2.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/icons/125.bmp</normaloff>:/images/icons/125.bmp</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
   <property name="toolTip">
    <string>项选择</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actSelPopMenu</sender>
   <signal>triggered()</signal>
   <receiver>actSelInvs</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
